packages/flutter_tools/lib/src/android/android_device.dart

```diff
@@ -519,7 +519,7 @@ class AndroidDevice extends Device {
 
   @override
   Future<LaunchResult> startApp(
-    AndroidApk package, {
+    AndroidApk? package, {
```

Comment:
Why allow null for Android app?

Reply:
It should be nullable because the base class is nullable due to web.  That's the bug this is fixing--`covariant` was hiding that null could be passed in, which in some cases caused crashes.

Reply:
yup, that was a brain fart on my part